Gems/PhysX/Code/Tests/EditorMeshColliderComponentTests.cpp

```diff
@@ -147,30 +147,285 @@ namespace PhysXEditorTests
         }
 
         // create an editor entity with a collider component
-        EntityPtr editorEntity = CreateMeshColliderEditorEntity(
-            AZ::Transform::CreateIdentity(), meshAssetData->CreateMeshAsset(), AZ::Vector3::CreateZero(), AZStd::nullopt, RigidBodyType::Dynamic);
+        EntityPtr editorEntity = CreateMeshColliderEditorEntity(meshAssetData->CreateMeshAsset());
 
         EntityPtr gameEntity = CreateActiveGameEntityFromEditorEntity(editorEntity.get());
 
-        // since there was an editor rigid body component, the runtime entity should have a dynamic rigid body
-        const auto* rigidBody = gameEntity->FindComponent<PhysX::RigidBodyComponent>()->GetRigidBody();
-        const auto* pxRigidDynamic = static_cast<const physx::PxRigidDynamic*>(rigidBody->GetNativePointer());
+        // check that the runtime entity has the expected components
+        auto* meshColliderComponent = gameEntity->FindComponent<PhysX::MeshColliderComponent>();
+        auto* staticRigidBodyComponent = gameEntity->FindComponent<PhysX::StaticRigidBodyComponent>();
+        EXPECT_TRUE(meshColliderComponent != nullptr);
+        EXPECT_TRUE(staticRigidBodyComponent != nullptr);
 
-        PHYSX_SCENE_READ_LOCK(pxRigidDynamic->getScene());
+        AzPhysics::ShapeColliderPairList shapeConfigList = meshColliderComponent->GetShapeConfigurations();
+        EXPECT_EQ(shapeConfigList.size(), 1);
+
+        for (const auto& shapeConfigPair : shapeConfigList)
+        {
+            EXPECT_EQ(shapeConfigPair.second->GetShapeType(), Physics::ShapeType::PhysicsAsset);
+        }
+
+        const auto* simulatedBody = staticRigidBodyComponent->GetSimulatedBody();
+        const auto* pxRigidStatic = static_cast<const physx::PxRigidStatic*>(simulatedBody->GetNativePointer());
+
+        PHYSX_SCENE_READ_LOCK(pxRigidStatic->getScene());
 
         // there should be a single shape on the rigid body and it should be a box
-        EXPECT_EQ(pxRigidDynamic->getNbShapes(), 1);
-        if (pxRigidDynamic->getNbShapes() > 0)
+        EXPECT_EQ(pxRigidStatic->getNbShapes(), 1);
+        if (pxRigidStatic->getNbShapes() > 0)
         {
             physx::PxShape* shape = nullptr;
-            pxRigidDynamic->getShapes(&shape, 1, 0);
+            pxRigidStatic->getShapes(&shape, 1, 0);
             EXPECT_EQ(shape->getGeometryType(), physx::PxGeometryType::eSPHERE);
         }
 
+        const AZ::Aabb aabb = simulatedBody->GetAabb();
+        EXPECT_THAT(aabb.GetMin(), UnitTest::IsCloseTolerance(AZ::Vector3(-0.5f, -0.5f, -0.5f), 1e-3f));
+        EXPECT_THAT(aabb.GetMax(), UnitTest::IsCloseTolerance(AZ::Vector3(0.5f, 0.5f, 0.5f), 1e-3f));
+
+        delete meshAssetData;
+    }
+
+    // [GHI-14907]
```

Comment:
I prefer to use the Github notation for issues since it provides a clear standard and expectation for others familiar with Github issue notation.

EG: `o3de/o3de#14907` or `o3de#14907`

https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/autolinked-references-and-urls#issues-and-pull-requests

Reply:
All the references to github issues in C++ is done using GHI-xxxx (mostly from the the [deprecation notices](https://github.com/o3de/community/blob/main/guides/o3de-deprecation-guidelines.md#api-deprecation---initial-steps)). I did a quick search and `o3de/o3de#xxxx` format is used in python files.

Since this is not a deprecation notice I'll change it since the format you suggest provides more accurate info.